main.py

```diff
@@ -417,7 +451,10 @@ async def play_next_song(skip_count: int = 0) -> None:
 
         # Say our goodbyes
         embed_title = "❤️‍🔥 That's it everyone ❤️‍🔥"
-        embed_content = "Hope ya had a good **BUST!**"
+        embed_content = "*Total length of all submissions: {}*\n".format(
+            format_time(int(total_song_len))
+        )
```

Comment:
What do you think about putting the "total length" text at the bottom of the end message instead?